.github/workflows/updater.yml

```diff
@@ -1,35 +1,64 @@
 name: Check Virtualenv
 on:
-  # schedule:
-  #   - cron: "0 0 * * *"
-  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+  #workflow_dispatch:
 
 jobs:
-  check-for-updates:
+  publish:
+    name: Publish new tag
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        versionRegex:
+         - v2\.[0-9]+\.[0-9]+$
+
```

Comment:
Great! Thank you @killmeplz , plz remove matrix, we don't plan support for multiple runner versions, the latest runner version is only relevant...